config/env/default.js

```diff
@@ -14,7 +14,7 @@ module.exports = {
     // session expiration is set by default to 24 hours
     maxAge: 24 * (60 * 60 * 1000),
     // httpOnly flag makes sure the cookie is only accessed
-    // through the HTTP protocol and not JS/browser 
+    // through the HTTP protocol and not JS/browser
```

Comment:
Some weird whitespace changes snuck in - if you're bored and want to nix them :P